lib/AgrirouterClient/CMakeLists.txt

```diff
@@ -22,8 +22,11 @@ file(GLOB SRCS
     "inc/Protobuf/messaging/response/payload/account/endpoints.pb.cc"
     "inc/Protobuf/messaging/response/payload/endpoint/capability.pb.cc"
     "inc/Protobuf/messaging/response/payload/feed/feed-response.pb.cc"
+    "inc/Protobuf/messaging/response/payload/feed/push-notification.pb.cc"
     "inc/Protobuf/messaging/response/response.pb.cc"
+    "inc/Protobuf/settings/dh-settings.pb.cc"
```

Comment:
dh-settings.proto is deprecated and won't be used in the project at all as far as i know.
The file should be reomved.